policies/aspnetcore.rego

```diff
@@ -0,0 +1,58 @@
+package tickets.aspnetcore
+
+import rego.v1
+
+default main := {"decision": false}
+
+auth_header := input.action.headers.Authorization
+
+tenant := trim(split(split(auth_header, "/")[0], " ")[1], " \t")
+
+user := trim(split(auth_header, "/")[1], " \t")
+
+action := a if {
+	input.action.name == "GET"
+	input.resource.id == "/api/tickets"
```

Comment:
[policy change]: We get the full URL from the framework, not a truncated one. (Example: `/api/tickets` vs `/tickets`)